src/main/resources/org/jpeek/xsd/matrix.xsd

```diff
@@ -26,6 +26,9 @@ SOFTWARE.
 @todo #134:30min Matrix.xsd: add `xsd:documentation` tags as per issue #134. We need to
  document the schemas in src/resources/org/jpeek/xsd so that the semantics of
  the generated XML documents is transparent for maintainers.
+@todo #135:30min Matrix.xsd: add a reference to this schema from the generated
+ skeleton.xml. Schemas from src/resources/org/jpeek/xsd must be referenced in
```

Comment:
@llorllale what `skeleton.xml` has to do with this file? Typo?

Reply:
@yegor256 yes, typo